app/views/products/_form.html.haml

```diff
@@ -0,0 +1,29 @@
+%div.container
+  %div.row.justify-content-center
+    %div.col-10
+      =bootstrap_form_with model: @product, class: "shadow p-3 mb-3 bg-info rounded", local: true do |f|
+        =f.text_field :name, placeholder: "Name of product"
+        %div.row
+          %div.col
+            =f.file_field :image, accept: 'image/jpeg,image/jpg,image/png', id: "input"
+            -if @product.image.attached?
+              =link_to 'Delete image', active_storage_attachment_path(@product.image.id), method: :delete, remote: :true, id: "button", class: "btn btn-outline-light"
```

Comment:
does rubocop inspect these files? we need to set maximum line length at 120 characters

Reply:
For some reasons RuboCop don't inspect views, I have no idea why